### DISCUSSION TITLE
directly download an anime episode with just one command

### DISCUSSION THREAD

**Discussion by Quicksilver151** [pystardust/ani-cli#862] [Ideas] [answered]
i want to be able to integrate another tool with ani-cli
would it be possible to have ani-cli download an anime with something like:

`ani-cli -d [anime_name/anime_link] [episode_number]`
then it just downloads that episode from that anime specifically

eg: `ani-cli -d "classroom of the elite season 2" 2`
result: downloads episode 2 of classroom of the elite season 2

**Comment by CoolnsX** [comment, marked as answer]
```ani-cli -d -a2 "classroom of the elite season 2"```
will download 2nd episode
```ani-cli -d -a-1 "classroom of the elite season 2"```
will download the last/latest released episode
